styles/styles.css

```diff
@@ -157,8 +158,18 @@ body.no-scroll {
   overflow: hidden;
 }
 
-main .section .default-content-wrapper { /* mobile */
-  padding: 0 var(--body-padding);
+.section {
```

Comment:
Let use `main .section` and just set the padding and max-width and adapt it to media queries.